pkg/virt-handler/non-root.go

```diff
@@ -122,6 +122,32 @@ func (d *VirtualMachineController) prepareTap(vmi *v1.VirtualMachineInstance, re
 
 }
 
+func (*VirtualMachineController) prepareVFIO(vmi *v1.VirtualMachineInstance, res isolation.IsolationResult) error {
+	vfioPath := filepath.Join(res.MountRoot(), "dev", "vfio")
+	err := os.Chmod(filepath.Join(vfioPath, "vfio"), 0666)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil
```

Comment:
I think its worth logging when /dev/vfio/vfio is not found.

Reply:
Right now we don't actually check if the device should be there and execute this function always(at least in the case of nonroot launcher). I think there is no way that the device would be missing. The launcher will exit with a meaningful message on its own if it would be missing.